google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/CloudSdkUtil.java

```diff
@@ -117,6 +117,10 @@ public static String toSdkHomeDirectory(String sdkExecutablePath) {
     return null;
   }
 
+  public static String getSystemCommand() {
```

Comment:
Nevermind I could easily mock SystemInfo by adding an adapter in SystemEnvironmentProvider (that's why I made this class anyway).